commit 0aa5b28a504c6f1f17b387d8147715d1496fff62
Author: Joe Simmons-Talbott <josimmon@redhat.com>
Date:   Fri Apr 21 09:24:25 2023 -0400

    wcsmbs: Add wcsdup() tests. (BZ #30266)
    
    Enable wide character testcases for wcsdup().
    
    Reviewed-by: Adhemerval Zanella  <adhemerval.zanella@linaro.org>
    Conflicts:
	wcsmbs/Makefile
          (different test backport order)


diff -Nrup a/wcsmbs/Makefile b/wcsmbs/Makefile
--- a/wcsmbs/Makefile	2023-11-30 14:14:18.755010508 -0500
+++ b/wcsmbs/Makefile	2023-11-30 14:38:18.511131851 -0500
@@ -53,7 +53,8 @@ tests := tst-wcstof wcsmbs-tst1 tst-wcsn
 	 tst-c16c32-1 wcsatcliff tst-wcstol-locale tst-wcstod-nan-locale \
 	 tst-wcstod-round test-char-types tst-fgetwc-after-eof \
 	 tst-wcstod-nan-sign tst-c16-surrogate tst-c32-state \
-	 $(addprefix test-,$(strop-tests)) tst-mbstowcs
+	 $(addprefix test-,$(strop-tests)) tst-mbstowcs \
+	 test-wcsdup
 
 include ../Rules
 
diff -Nrup a/wcsmbs/test-wcsdup.c b/wcsmbs/test-wcsdup.c
--- a/wcsmbs/test-wcsdup.c	1969-12-31 19:00:00.000000000 -0500
+++ b/wcsmbs/test-wcsdup.c	2023-11-30 14:14:48.869138712 -0500
@@ -0,0 +1,2 @@
+#define WIDE 1
+#include "../string/test-strdup.c"
